src/components/homepage-users.module.scss

```diff
@@ -46,7 +46,7 @@
   text-align: center;
 
   &::before {
-    content: '';
+    content: "";
```

Comment:
we should avoid making unnecessary code style changes outside of code style PRs to limit conflicts

Reply:
Does eslint run on scss as well? If so; can we make 1 big PR to do all style changes according to what we use for the main repo as well? If not; can we enable that and then do the same?

Reply:
It doesn't but we could set up stylelint. I don't know if it's well integrated in IDEs though. Maybe using prettier for scss would work better

Reply:
That might be better, @sdepold do you want to look into this?

Reply:
Yep